gulpfile.js

```diff
@@ -493,9 +512,17 @@ const buildAllNoDeveloperTools = ((cache) =>
     )(done);
   }))({});
 
+const compileDeveloperTools = ((cache) =>
+  memoizeTask(cache, function compileDeveloperTools(format, done, type) {
+    return gulp.parallel(
+      compileDeveloperToolDeclarations(format, type),
+      compileDeveloperToolsBundle(format, type),
```

Comment:
`type` is passed to `compileDeveloperToolsBundle`, but `compileDeveloperToolsBundle` does not require a `type` argument.